src/indra_cogex/sources/clinicaltrials/__init__.py

```diff
@@ -95,7 +55,16 @@ def ground_drug(self, drug):
         return None
 
     def get_nodes(self):
-        for index, row in tqdm.tqdm(self.df.iterrows(), total=len(self.df)):
+        nctid_to_data = {}
+        for _, row in tqdm.tqdm(self.df.iterrows(), total=len(self.df)):
+            nctid_to_data[row["NCTId"]] = {
+                "study_type": row["StudyType"],  # observational, interventional
```

Comment:
There is a small but significant amount of rows (I count 914) that also have 'Expanded Access' as StudyType.